Main/ClientApp/src/Api/ConfigurationApi.ts

```diff
@@ -0,0 +1,22 @@
+import {
+  authorizedGet,
+  authorizedPatch,
+} from "../Infrastructure/FetchUtilities";
+
+class ConfigurationApi {
+  async getConfiguration(): Promise<Map<string, string>> {
+    let response = await authorizedGet("api/configuration");
+    let configurationArr = (await response.json()) as string[][];
+    let map = new Map();
+    configurationArr.forEach((a) => map.set(a[0], a[1]));
```

Comment:
Please create a proper Configuration class instead of keeping all of this in a dictionary.